mdnDocsOverrides.json

```diff
@@ -19,31 +19,45 @@
     "Web/CSS/abs",
     "Web/CSS/sign"
   ],
-  "absolute-positioning": [],
+  "absolute-positioning": [
+    "Web/CSS/position"
+  ],
   "accelerometer": [
     "Web/API/Accelerometer"
   ],
   "accent-color": [
     "Web/CSS/accent-color"
   ],
-  "active-view-transition": [],
+  "active-view-transition": [
+    "Web/CSS/::view-transition"
+  ],
   "address": [
     "Web/HTML/Element/address"
   ],
-  "alerts": [],
-  "align-content-block": [],
-  "alignment-baseline": [],
+  "alerts": [
+    "Web/API/Window/alert"
+  ],
+  "align-content-block": [
+    "Web/CSS/align-content"
+  ],
+  "alignment-baseline": [
+    "Web/SVG/Attribute/alignment-baseline"
```

Comment:
I would have expected this to be under CSS, instead of under SVG. The description of the feature doesn't say anything about SVG. Let's leave it like this for now.